projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -502,7 +502,7 @@ private function is_valid_admin_url_or_absolute_path( $input ) {
 		}
 
 		// Require that the string start with a slash, but not two slashes.
-		if ( '/' === substr( $input, 0, 1 ) && '/' !== substr( $input, 1, 1 ) ) {
+		if ( str_starts_with( $input, '/' ) && '/' !== substr( $input, 1, 1 ) ) {
```

Comment:
Maybe do this? I'm not sure.
```suggestion
		if ( str_starts_with( $input, '/' ) && ! str_starts_with( $input, '//' ) ) {
```

Reply:
Yeah, why not? 👍